### ISSUE TITLE
Deleted Albums in Spotify are not deleted in Music Assistant

### ISSUE THREAD

**Issue by Fatalmose** [music-assistant/hass-music-assistant#2835]
### What version of Music Assistant has the issue?

2.3.0B7

### What version of the Home Assistant Integration have you got installed?

2024.8.1

### Have you tried everything in the Troubleshooting FAQ and reviewed the Open and Closed Issues and Discussions to resolve this yourself?

- [X] Yes

### The problem

I deleted all my albums in Spotify.
So now I have a clean Spotify library with nothing in it so we can make a fresh start.
After synchronization with Spotify , all of my previous deleted albums are coming back in Music Assistant library or are not deleted.
I took the time to delete them all within MA itself  , but after the next sync. they all came back.
So now I have an empty Spotify and 625 albums showing in MA which I can not be deleted.

### How to reproduce

Delete your albums and playlists in Spotify.
Synchronize MA with Spotify.
Delete Albums in MA itself.
Synchronize again.

### Music Providers

Spotify.

### Player Providers

Airplay.
Chromecast.
UPnP / DLNA Player.
Universal Group player.

### What version of Home Assistant Core are your running

2024.8.1

### What type of installation are you running?

Home Assistant OS

### On what type of hardware are you running?

Raspberry Pi

**Comment by Fatalmose** [comment]
I found the solution.
When playlists or albums are liked in Spotify they can not be deleted in MA.
Every time you sync. Spotify within MA it will automatically generate the liked albums and playlists in MA music library.

**Comment by OzGav** [comment]
Hmmm… but removing them from the MA library should have also removed them from the Spotify library. I just tried this and it worked fine!

**Comment by Fatalmose** [comment]
Hello.

Thanks for the reply.
I figured out that most of my albums were liked in Spotify.
It were those who still got back in MA after sync.
I found a way to unlike all of them in one shot and after a MA/Spotify
sync. they were all gone.
If songs or albums in Spotify are liked they are kept in the liked playlist
even when you delete them from your Spotify library.
So I think that when MA makes sync. with Spotify it downloads in it’s
library all of the likes songs or albums even if you have an empty Spotify
library.
Although that’s what I noticed.

Regards.

JS.
